### DISCUSSION TITLE
Sample app

### DISCUSSION THREAD

**Discussion by danrald** [skeletonlabs/skeleton#563] [Q&A] [answered]
Im new to svelte and tailwind. Is there a demo app? I read the directions for a new build, got one going. But I cant seem to get the styling in properly. 

Thanks

**Comment by endigo9740** [comment, marked as answer]
Hi @danrald The documentation site (this repo) technically serves as a demo app, it's built with Skeleton. Though the library and docs are part of the same SvelteKit project, so paths will be relative rather than referring to the package in node_modules. Additionally we're doing some funky stuff with the styles to support all the themes and theme switching. So it may be useful for everything but styles.

We also have written guides for creating a simple project per each of the three supported frameworks. These are step-by-step guides that should guide you through most of the process:
https://www.skeleton.dev/guides/frameworks

Finally, we have a CLI tool in development for SvelteKit projects that will soon automatically scaffold a project for you with the latest dependencies in place. We're leaning into this instead maintaining separate demo apps per each framework:
- https://github.com/Brain-Bones/skeleton/issues/552

There's three common problems we often see related to styling for new users:

1. Not importing all the styles needed (theme, all.css, your global stylesheet)
2. Importing these in the incorrect order (above is correct), which is detailed here: 
https://www.skeleton.dev/guides/styling
3. Failing to delete the @ tailwind directives that are auto-generated in your global stylesheet as part of Svelte-Add. This is detailed on the same page linked above.

If none of this helps let me know and I'd be glad to troubleshoot further. I'm online on Discord right now (@endigo9740) if you want to reach out in our #support channel

**Reply by danrald** [reply]
Oh Wow! 
Thanks, I just got the code for this repo, did an npm install then 
npm run dev .. then whamo! It worked. Thank you very much. I go over the code!

**Reply by endigo9740** [reply]
Let me know if we can clarify anything. Again a reminder this repo differs a bit from our recommended instructions for end users. But probably will some useful info and techniques to garner!

**Reply by endigo9740** [reply]
Also feel free to hit the "mark as answer" button above if you feel this answers your original question please. Thanks!
